lib/client-assets.php

```diff
@@ -550,7 +551,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	);
 
 	// Prepopulate with some test content in demo.
-	if ( $is_demo ) {
+	if ( $is_new_post && $is_demo ) {
```

Comment:
It's kind of odd that demo is a new post and also a demo. But it may also be a way to initialize any post with existing content, which could be great for ready-made templates.